cli/tests/pcluster/templates/test_cluster_stack.py

```diff
@@ -364,25 +366,40 @@ def get_generated_template_and_cdk_assets(
                 InstanceTypeLTAssertion(has_instance_type=True),
                 EbsLTAssertion(includes_ebs_optimized=True, is_ebs_optimized=True),
             ],
+            False,
+            "",
         ),
     ],
 )
 def test_compute_launch_template_properties(
     mocker,
     config_file_name,
     lt_assertions,
+    disable_sudo_access_default_user,
+    expected_cloud_config_for_default_user,
+    pcluster_config_reader,
     test_datadir,
 ):
+    rendered_config_file = pcluster_config_reader(
+        config_file_name,
+        disable_sudo_access_default_user=disable_sudo_access_default_user,
+    )
     generated_template, cdk_assets = get_generated_template_and_cdk_assets(
         mocker,
-        config_file_name,
+        rendered_config_file,
         test_datadir,
     )
 
     asset_content = get_asset_content_with_resource_name(cdk_assets, "LaunchTemplate64e1c3597ca4c326")
 
     for lt_assertion in lt_assertions:
         lt_assertion.assert_lt_properties(asset_content, "LaunchTemplate64e1c3597ca4c326")
+    # Checking for DisableSudoAccessForDefaultUser
+    assert_that(
+        asset_content["Resources"]["LaunchTemplate64e1c3597ca4c326"]["Properties"]["LaunchTemplateData"]["UserData"][
```

Comment:
The launch template logical id (LaunchTemplate64e1c3597ca4c326) is repeat three times, what about defining a constant?